src/ThemeProvider/index.tsx

```diff
@@ -59,8 +39,29 @@ declare module '@emotion/react' {
   }
 }
 
```

Comment:
these two `interface Theme` are distinctly scoped

Reply:
one is needed for the local uikit project to declaration merge with @emotion,
the second export one is needed for declaration merging in a consume project to work